util/generate_dict/generate_private_dict.py

```diff
@@ -24,6 +24,55 @@
 """
 
 
+# Manual additions for the autogenerated dict
+ADDITIONS = {
+    "ELSCINT1": {
+        # Philips Access CT DICOM Conformance Statement (May 2015)
+        "00E11021": ("DS", "1", "DLP", ""),
+        "00E11037": ("DS", "1", "Total Saving Dose", ""),
+        "00E1103E": ("IS", "1", "Split Is Dual Surview", ""),
+        "00E11050": ("DS", "1", "Acquisition Duration", ""),
+        "00E110C4": ("DS", "1", "Abs Bed Pos", ""),
+        "01E11017": ("UI", "1", "ECG Reference UID", ""),
+        "01E11026": ("CS", "1", "Phantom Type", ""),
+        "01F11001": ("CS", "1", "Acquisition Type", ""),
+        "01F11002": ("CS", "1", "Resolution", ""),
+        "01F11004": ("CS", "1", "Angular Sampling", ""),
+        "01F11008": ("DS", "1", "Scan Length", ""),
+        "01F1100C": ("DS", "1", "Scanner Relative Center", ""),
+        "01F1100E": ("FL", "1", "Recon Enhancement", ""),
+        "01F11026": ("FD", "1", "Pitch", ""),
+        "01F11027": ("DS", "1", "Rotation Time", ""),
+        "01F11028": ("DS", "1", "Table Increment", ""),
+        "01F11032": ("CS", "1", "View Convention", ""),
+        "01F11033": ("DS", "1", "Cycle Time", ""),
+        "01F11041": ("LO", "1", "Gating Delay", ""),
+        "01F11045": ("IS", "1", "Initial Heart Rate", ""),
+        "01F11049": ("DS", "1", "Planned mAs", ""),
+        "01F1104B": ("SH", "1", "Collimation", ""),
+        "01F1104C": ("SH", "1", "DOSE Right DOM", ""),
+        "01F1104D": ("SH", "1", "Adaptive Filter", ""),
+        "01F1104E": ("SH", "1", "Scan Type", ""),
+        "01F7109B": ("IS", "1", "iDose Level", ""),
+        "01F91001": ("LO", "1", "Mar Filter", ""),
+        "01F91002": ("DS", "1", "Recon Increment", ""),
+        "01F91003": ("DS", "1", "CTDIw", ""),
+        "01F91004": ("IS", "1", "Couch Direction", ""),
+        "01F91005": ("IS", "1", "Series No In Acquisition", ""),
+        "01F91007": ("SH", "1", "Dose Right ACS", ""),
+        "01F91008": ("DS", "1", "Left DMS Tmp Diff", ""),
+        "01F91009": ("DS", "1", "Right DMS Tmp Diff", ""),
+        "01F91010": ("LO", "1", "Dose Right Noise", ""),
+        "01F91011": ("DS", "1", "Zero Position", ""),
+        "01F91012": ("DS", "1", "Show Couch Position", ""),
+        "01F91013": ("IS", "1", "Recon Mode", ""),
+        "01F91014": ("DS", "1", "Water Size", ""),
+        "01F91015": ("DS", "1", "Digital Tilt", ""),
+        "01F91016": ("DS", "1", "Scan Arc", ""),
+    },
```

Comment:
It would probably make sense to create a PR in [GDCM](https://github.com/malaterre/GDCM/blob/master/Source/DataDictionary/privatedicts.xml) for this, so this can be removed again later.